src/app/app.component.html

```diff
@@ -2,13 +2,6 @@ <h1>{{title}}</h1>
 <!-- Needs Router-outlet for Router to change page-->
 <router-outlet></router-outlet>
 
-<ul>
-  <li class="text" *ngFor="let item of items | async">
-    {{item.name}}
-  </li>
-</ul>
-<button mat-raised-button>This is a mat button!</button>
-
```

Comment:
Where did you move this?

Reply:
I removed it as it was test code I used to see if Firebase was working. It had no purpose.